data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
+mission "Remnant: D94-YV Shield Generator"
+	name "Retrieve D94-YV Shield Generators for the Remnant"
+	description "A Remnant engineer has offered you <payment> for delivering two D94-YV Shield Generators to a research facility on <planet>."
+	minor
+	source "Viminal"
+	destination
+		government "Remnant"
+	to offer
+		or
+			has "Remnant: Tech Retrieval: active"
+			has "Remnant: Tech Retrieval: done"
+	on offer
+		require "D94-YV Shield Generator"
+		conversation
+			`You recall that Taely was interested in seeing examples of new human technology, and the D94-YV Shield Generator probably fits the criteria. Would you like to show her?`
 			choice
-				`	"Sure, I would be glad to work out a deal."`
-				`	"Sorry, I'm really not interested in helping Remnant research."`
-					decline
+				`	Not now.`
+					defer
+				`	Yes.`
+			branch stuff
+				has "Remnant Tech Retrieval: done"
+			`	After asking a few people, you finally get directed to a bay in the shipyard where you find Taely applying some kind of material over a hole in a hull. "Greetings, <first>. What brings you here today?" She reseals the hatch, and drops to the ground on a tether. "Did you bring us something?"`
+				goto the_rest
+			label stuff
+			`	Remembering that she spends most of her time in the shipyard, you head straight there. As you search through the yard you are soon approached by another engineer. "Well met, Captain. Taely said you may be bringing back samples of new technology from ancestral space. Do you have something to show us?`
+			label the_rest
+			`	You nod in the affirmative and lead her back to your ship, where she examines the D94-YV Shield Generator. "Huh, it certainly looks impressive," she comments as she paces around the massive piece of equipment. "It will be valuable to see what our enemies might be shielded with, and maybe something we can learn from it too." She comes to a stop in front of you. "If you could deliver two of these to a lab on <planet> we could compensate you with <payment>.`
+				accept
+	on complete
+		outfit "D94-YV Shield Generator" -2
+		payment 1150000
+		"remnant met taely" ++
+		conversation
+			`On <planet> you follow the landing instructions to arrive at a pad set next to a large warehouse. As you complete your shutdown routine the doors slide open and a large crawler emerges, accompanied by several remnant. They carefully unload the D94-YV Shield Generators and strap them down. As they start heading back to the warehouse one of them comes over to hand you <payment>. "Thank you for the delivery. Our team is eager to see what humanity has achieved in these past centuries."`
+
+
 
-			`	After a bit of negotiation, he agrees to pay you <payment> in exchange for two plasma cannons.`
+mission "Remnant: S-970 Regenerator"
+	name "Retrieve S-970 Regenerators for the Remnant"
```

Comment:
Name looks dangerously long.

Reply:
Removed the `for the Remnant` part. (Also removed it from the other 5 missions too)